cmd/root.go

```diff
@@ -61,11 +61,15 @@ func initConfig() {
 	config.LoadConfig(configPath, false)
 	log.ConfigureLogger(config.GetConfig().LogLevel, config.GetConfig().LogFormat, config.GetConfig().LogTimestamp)
 
-	if config.GetConfig().HTTPPort != "" {
-		split := strings.Split(config.GetConfig().HTTPPort, ":")
+	if len(config.GetConfig().HTTPPorts) != 0 {
+		split := strings.Split(config.GetConfig().HTTPPorts[0], ":")
+
+		lastIdx := len(split) - 1
+
+		apiHost = strings.Join(split[:lastIdx], ":")
```

Comment:
This isn't completely related, but I added this behavior as I noticed the discrepancy while fixing the code to work with a list.
I can remove commit 79dcf57e6689fbbdbe85ef45a8c19abc2988c69d if you prefer.